client/my-sites/patterns/components/search-field/index.tsx

```diff
@@ -27,6 +27,11 @@ export const PatternsSearchField = ( { isCollapsible = false }: PatternsSearchFi
 			url.searchParams.delete( QUERY_PARAM_SEARCH );
 		}
 
+		// Blur the input field when the search form is submitted
+		if ( document.activeElement instanceof HTMLInputElement ) {
+			document.activeElement.blur();
```

Comment:
> When submitting the search form, the input is blurred (this goes for both input fields).

Oh, here you meant - "blur" as action, and I thought that "make it disabled / in progress" when search is happening 😄